configs/face/2d_kpt_sview_rgb_img/topdown_heatmap/cofw/hrnetv2_cofw.md

```diff
@@ -0,0 +1,38 @@
+<!-- [ALGORITHM] -->
+
+<details>
+<summary align="right">HRNetv2 (TPAMI'2019)</summary>
+
+```bibtex
+@article{WangSCJDZLMTWLX19,
+  title={Deep High-Resolution Representation Learning for Visual Recognition},
+  author={Jingdong Wang and Ke Sun and Tianheng Cheng and
+          Borui Jiang and Chaorui Deng and Yang Zhao and Dong Liu and Yadong Mu and
+          Mingkui Tan and Xinggang Wang and Wenyu Liu and Bin Xiao},
+  journal={TPAMI},
+  year={2019}
+}
+```
+
+</details>
+
+<!-- [DATASET] -->
+
+<details>
+<summary align="right">COFW (ICCV'2013)</summary>
+
+```bibtex
+@inproceedings{burgos2013robust,
+  title={Robust face landmark estimation under occlusion},
+  author={Burgos-Artizzu, Xavier P and Perona, Pietro and Doll{\'a}r, Piotr},
+  booktitle={Proceedings of the IEEE international conference on computer vision},
+  pages={1513--1520},
+  year={2013}
+}
+```
+
+</details>
+
+</details>
+
+Results on COFW dataset
```

Comment:
Add a note for these empty slots: checkpoints will be revealed after mmpose reach 1k stars 😃